ni/protobuf/types/waveform.proto

```diff
@@ -43,6 +43,106 @@ message DoubleAnalogWaveform
   map<string, WaveformAttributeValue> attributes = 4;
 }
 
+// I16AnalogWaveform datatype.
+message I16AnalogWaveform {
+  // The trigger time of the waveform.
```

Comment:
This is not precisely correct. The t0 field is the time of the first sample in y_data, which may be offset from the trigger time. (For example, DAQmx has StartTrig.Delay or whatever, and also reference triggers have a negative delay corresopnding to PreTriggerSamples.)